### DISCUSSION TITLE
Round pixels for anchor

### DISCUSSION THREAD

**Discussion by gewisser** [pixijs/pixijs#7844] [Q&A] [answered]
Hey! Who can tell you why roundPixels for anchor was not implemented? Below are 2 pictures:

![image](https://user-images.githubusercontent.com/5417292/135764655-658ed3bb-d6bc-4c29-afb5-ecebaec23976.png)

top image: 
```javascript
const label = new PIXI.BitmapText(this.pix2x(x_).toFixed(2), {
  fontName: "Arial",
  fontSize: -12,
});

label.anchor.set(0.5, 0);
```

bottom image:
```javascript
const label = new PIXI.BitmapText(this.pix2x(x_).toFixed(2), {
  fontName: "Arial",
  fontSize: -12,
});

label.anchor.set(0, 0);
```

**Comment by ivanpopelyshev** [comment, marked as answer]
That's the only thing where `roundPixels` is used in `BitmapText`. 
https://github.com/pixijs/pixijs/blob/cf828cf40c36a6d8cd2af7c4a79287238f3437b9/packages/text-bitmap/src/BitmapText.ts#L526-L529

BitmapText is work-in-progress class, many people are patching it. Two ways how it goes:
1. you make a PR, with tests
2. someone else interested (from pixijs team or just collaborator) will do it.

**Reply by gewisser** [reply]
Thank you Ivan. Unfortunately, my experience is not enough to efficiently implement this functionality. So far I have left this code for myself:
```javascript
const label = new PIXI.BitmapText(this.pix2x(x_).toFixed(2), {
  fontName: "Arial",
  fontSize: -12,
  tint: 0xff2525,
});

const offset = Math.floor(label.textWidth / 2);
label.x = x_ - offset;
label.y = y1 + 10;
```

**Reply by ivanpopelyshev** [reply]
ok, but if you make at least example (jsfiddle, codepen or pixi-playground) we can create issue on that :) then somebody else will make PR
